scripts/lib/CIME/test_scheduler.py

```diff
@@ -526,8 +528,27 @@ def _xml_phase(self, test):
                     envtest.set_test_parameter("STOP_OPTION",stop_option[opt])
                     opti = match.group(2)
                     envtest.set_test_parameter("STOP_N", opti)
-                    logger.debug (" STOP_OPTION set to {}".format(stop_option[opt]))
-                    logger.debug (" STOP_N      set to {}".format(opti))
+
+                    logger.debug (" STOP_OPTION set to %s" %stop_option[opt])
```

Comment:
These need to be converted back to python3-style.

Reply:
Once you have completed testing with python2 and resolved the concerns raised in review I will run testing with python3.